build/repo.props

```diff
@@ -25,6 +25,9 @@
     <DisableSignCheckStrongName>true</DisableSignCheckStrongName>
 
     <SharedSourcesFolder>$(RepositoryRoot)src\Shared\</SharedSourcesFolder>
+    <SharedFxArchitecture Condition="'$(SharedFxArchitecture)' == ''">$(SharedFxRid.Substring($([MSBuild]::Add($(SharedFxRid.LastIndexOf('-')), 1))))</SharedFxArchitecture>
+    <BuildSiteExtension>false</BuildSiteExtension>
```

Comment:
@pakrym The build is failing right because of missing Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.0.x64 artifact. The reason for that is the artifacts.props uses the `BuildSiteExtensions` property while the target that builds the package is conditional on `BuildSiteExtension` (plural vs singular). Which do you prefer?

Reply:
Entire site extension commit should be no-op merged. It's already implemented in 3.0 in different way.